transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java

```diff
@@ -39,25 +39,29 @@
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
+import com.hyperwallet.android.insight.InsightEventTag;
 import com.hyperwallet.android.model.HyperwalletError;
 import com.hyperwallet.android.ui.R;
+import com.hyperwallet.android.ui.common.insight.HyperwalletInsight;
 import com.hyperwallet.android.ui.common.view.HorizontalDividerItemDecorator;
 import com.hyperwallet.android.ui.common.view.OneClickListener;
 import com.hyperwallet.android.ui.transfermethod.repository.TransferMethodRepositoryFactory;
 import com.hyperwallet.android.ui.user.repository.UserRepositoryFactory;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
+import java.util.Map;
 import java.util.TreeMap;
 
 public class SelectTransferMethodFragment extends Fragment implements SelectTransferMethodContract.View {
 
+    protected static final String TAG = "transfer-method:add:select-transfer-method";
+    private static final String LINK = "select-country";
```

Comment:
@peter-joseph This US7(PR-112) LINK constant name may conflict with US3US7(PR-111).

Reply:
the name can be adjusted that is why it's not mentioned in the ticket or design on what name you want to call the constants for link, obviously in fragment there can be more links on it
so in this case it can be like `LINK_SELECT_COUNTRY`